packages/wcwidth/wcwidth.1.0.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "Get terminal display width of a string"
+description: "Get terminal display width of a string"
+maintainer: ["Jonathan Yong"]
+authors: ["Jonathan Yong"]
+license: "MIT"
+homepage: "https://github.com/yongrenjie/wcwidth-ocaml"
+bug-reports: "https://github.com/yongrenjie/wcwidth-ocaml/issues"
+depends: [
+  "dune" {>= "3.3"}
+  "odoc" {with-doc}
```

Comment:
```suggestion
  "alcotest" {with-test}
  "odoc" {with-doc}
```

The tests don't pass in the CI because the alcotest package is not installed. You might need to also add bounds on the version of alcotest you require… but you certainly need to declare the dependency.